### DISCUSSION TITLE
git: 'credential-manager-core' is not a git command.

### DISCUSSION THREAD

**Discussion by trojanobelix** [gitextensions/gitextensions#11406] [Q&A] [answered]
### Environment

- Git Extensions 4.2.1.17611
- Build b0c0b284876b63c52c77195ada82a85bfadbb083
- Git 2.43.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.25
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.21 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

Got this error message
**git: 'credential-manager-core' is not a git command.**
 when pushing to GitLab Server after update to 4.2.1

### Steps to reproduce

Update to 4.2.1
Push comitted changes

### Did this work in previous version of GitExtensions?

yes 4.2.0

**Comment by pmiossec** [comment]
I think that's not a GitExtensions bug of v4.2.1 but more because you have updated Git and 'credential-manager-core' has been renamed. See https://github.com/git-ecosystem/git-credential-manager/blob/main/docs/rename.md

**Comment by trojanobelix** [comment]
No, I have updated only the gitexctension program today. Yesterday I pushed to the server and everthing was fine.
git was not updated for weeks

btw: The Log does not contain any "credential-manager-core" command
btw2: I would be very nice to be able to search in the log window or copy the log window (to search with other programm (notepad++)

**Comment by gerhardol** [comment, marked as answer]
GE does not execute credential-manager directly, it is done by Git. (Cannot be included in the log.)
gcm is normally used to http repos.
The settings is normally set in the system configuration when installing Git, not included in the GE configuration of Git settings.

One way to fix you Git setting is to reinstall Git. You may have an override for a specific repo too.

**Comment by trojanobelix** [comment]
> You may have an override for a specific repo too.

Where? 
Reinstalling git  does not solve this problem. But it seems only a problem of this repro (works on the repo pushed yesterday)

Thanks.
btw: Great program! Thanks! :-)

**Comment by pmiossec** [comment]
> But it seems only a problem of this repro (works on the repo pushed yesterday)

That's why @gerhardol told you that "You may have an override for a specific repo too."

Check with command in this repo: `git config --list --local`

**Comment by trojanobelix** [comment]
@pmiosec
Thanks, But the output does not point me to a config to change

```
git config --list --local
core.repositoryformatversion=0
core.filemode=false
core.bare=false
core.logallrefupdates=true
core.symlinks=false
core.ignorecase=true
core.hidedotfiles=dotGitOnly
remote.origin.url=I:\TK_05_technical\xxxxxxxxx
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.master.remote=gitlab
branch.master.merge=refs/heads/master
remote.gitlab.url=https://git-sw.xxxxxxx
remote.gitlab.fetch=+refs/heads/*:refs/remotes/gitlab/*
```

[EDIT] fixed formatting

**Reply by vbjay** [reply]
```bash
git config --show-origin --get-regexp ^credential\.
```

**Reply by vbjay** [reply]
```
file:C:/Program Files/Git/etc/gitconfig	credential.helper manager
file:C:/Program Files/Git/etc/gitconfig	credential.https://dev.azure.com.usehttppath true
file:C:/Users/vbjay/.gitconfig	credential.helper manager
```
my output from the command.
